WORKSPACE

```diff
@@ -39,3 +40,12 @@ llvm_toolchain(
     name = "llvm_toolchain",
     llvm_version = "6.0.0",
 )
+
+http_archive(
+    name = "bazel_toolchains",
+    sha256 = "7938215442cc96b1a16809f4643db14af7d91c3ef1e98d39593bb46862204537",
+    strip_prefix = "bazel-toolchains-809c13c3a96966f63c7a43c586cb4e72589184b6",
+    urls = [
+        "https://github.com/bazelbuild/bazel-toolchains/archive/809c13c3a96966f63c7a43c586cb4e72589184b6.tar.gz",
```

Comment:
can we pin them to the latest release instead of this commit?

Reply:
Will do that in a separate PR.